balboa-service-jms/src/main/java/com/socrata/balboa/jms/BalboaJms.java

```diff
@@ -49,12 +65,14 @@ public static void main(String[] args) throws Exception
             System.exit(1);
         }
 
+        Properties p = new Properties();
+        p.load(BalboaJms.class.getClassLoader().getResourceAsStream("config/config.properties"));
```

Comment:
getResourceAsStream can return a null. If it does, it would be better if the message said "'config/config.properties' not found on the classpath" instead of NullPointerException.